hw/xbox/nv2a/swizzle.c

```diff
@@ -142,24 +160,23 @@ void unswizzle_box(
     }
 }
 
-void unswizzle_rect(
-    const uint8_t *src_buf,
-    unsigned int width,
-    unsigned int height,
-    uint8_t *dst_buf,
-    unsigned int pitch,
-    unsigned int bytes_per_pixel)
+void unswizzle_rect(const uint8_t *src_buf,
+                    unsigned int width,
+                    unsigned int height,
+                    uint8_t *dst_buf,
+                    unsigned int pitch,
+                    unsigned int bytes_per_pixel)
 {
-    unswizzle_box(src_buf, width, height, 1, dst_buf, pitch, 0, bytes_per_pixel);
+    unswizzle_box(
+        src_buf, width, height, 1, dst_buf, pitch, 0, bytes_per_pixel);
```

Comment:
I'd rather see the list of arguments split halfway through than moving them into the next line.

Reply:
Idk why clang-format does this sometimes but not others, but agree